README.md

```diff
@@ -6,37 +6,19 @@ There are two distinct ways of handling user identities when building an applica
 
 ## Client Registration
 
-The first step in either implementation is registering a new client id as a "Service" type in the [iTwin App Registration](https://developer.bentley.com/my-apps/) and add the 'Visualization' API Association. This will provide a client id and client secret to use for getting an access token, via a [client credentials workflow](https://developer.bentley.com/apis/overview/authorization/#clientcredentialflow), to access the iTwin Platform.
+The first step in either implementation is registering a new client id as a "Service" type in the [iTwin App Registration](https://developer.bentley.com/my-apps/) and add the `Visualization` and `iModels` API Association. This will provide a client id and client secret to use for getting an access token, via a [client credentials workflow](https://developer.bentley.com/apis/overview/authorization/#clientcredentialflow), to access the iTwin Platform.
```

Comment:
Let's reword this the first line to "The first step is to register a new client is as a ...". Since we're removing the other way you could set this up.